tests/main/views/test_marketplace.py

```diff
@@ -326,7 +325,7 @@ def test_cookie_page(self):
         assert res.status_code == 200
         assert '<h1>Cookies</h1>' in self._strip_whitespace(res.get_data(as_text=True))
 
-    def test_cookie_page(self):
+    def test_ts_n_cs_page(self):
```

Comment:
I think a more explicit `test_terms_and_conditions_page` would be clearer here. Took me a few seconds to try to decipher ts_n_cs... :)